examples/ensemble/plot_stack_predictors.py

```diff
@@ -99,10 +99,7 @@ def load_ames_housing():
 
 cat_selector = make_column_selector(dtype_include=object)
 num_selector = make_column_selector(dtype_include=np.number)
-cat_selector(X)
```

Comment:
You let those two outputs. They are not costly to be evaluated. This is to show the name of the columns.